### DISCUSSION TITLE
Is there any way to set like `whichwrap` value in vim mode in zed editor?

### DISCUSSION THREAD

**Discussion by kyu-suke** [zed-industries/zed#22428] [Feature Requests]
I want to move the cursor from the beginning of a line to the end of the previous line when I pressed `h`. And move the cursor from the end of a line to the beginning of the next line when pressed `l`. In vimrc, I want to set set `whichwrap=b,s,h,l,<,>,[,]` or equivalent. Is it possible to make such a setting?

**Comment by kyu-suke** [comment]
Resolved. I use `vim::Space` and `vim::Backspace`

```
keymap.json
{
    "context": "vim_mode == normal",
    "bindings": {
      "l": "vim::Space",
      "h": "vim::Backspace"
    }
  }
```

**Comment by SGA-Yosuke-Takabatake** [comment]
This is already supported with the official `vim::WrappingLeft` / `vim::WrappingRight` bindings, so if you stumbled upon this thread, hope this helps!
https://github.com/zed-industries/zed/pull/25694
